### PR TITLE
File Added: Assignment 6 part-II

### PR DESCRIPTION
**Pull request by sourabh-kanwade**

### CODE REVIEW

File: Sourabh_Kanwade/html/Assignment 6/question_2.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#other {
+    margin: 20px 0;
+}
+.container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 70%;
+}
+
```
Comment (somesh6797) [inline Sourabh_Kanwade/html/Assignment 6/question_2.css]: remove space from here and push changes as soon as possible

**Review by somesh6797 — CHANGES_REQUESTED** [review]

File: Sourabh_Kanwade/html/Assignment 6/question_2.css (hunk 1 above)
Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 6/question_2.css]: done

**Review by shivling123 — APPROVED** [review]

**Review by somesh6797 — COMMENTED** [review]
All changes done..

**Review by somesh6797 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

File: Sourabh_Kanwade/html/Assignment 6/Question_2.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Question 2</title>
```
Comment (dayanandviveki) [inline Sourabh_Kanwade/html/Assignment 6/Question_2.html]: give proper title to assignments

**Review by dayanandviveki — COMMENTED** [review]
All code is ok but only change the title

**Review by dayanandviveki — APPROVED** [review]
No changes Required

**Review by kalyani1942 — APPROVED** [review]

File: Sourabh_Kanwade/html/Assignment 6/Question_2.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Form - Professional Details</title>
+        <link rel="stylesheet" href="./question_2.css" />
+    </head>
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>
+        <hr />
+        <h2>Personal Details</h2>
+        <label for="first-name">First Name: </label>
+        <input type="text" name="fname" id="first-name" placeholder="First Name" />
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 6/Question_2.html]: placeholders second words 1st letter should be small like First name make this change in all input boxes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

**Comment by sourabh-kanwade** [comment]
Today I did Assignment 7 so that changes were commited but not pushed to github . After making changes Assignment 6 Question 2.html those files of assignent 7 are also got added in PR . Please review them

File: Sourabh_Kanwade/html/Assignment 6/question_2.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+h1,
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 6/question_2.css]: include it in a single line h1, h2{}

File: Sourabh_Kanwade/html/Assignment 7/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    height: 40px;
+    background-color: #0f376b;
+    color: #fff;
+    font-weight: bold;
+    font-size: 15px;
+    outline: none;
+    border: none;
+    border-radius: 20px;
+}
+form > label,
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 7/css/styles.css]: include in a single line

File: Sourabh_Kanwade/html/Assignment 7/home.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        <title>Job Portal-Home</title>
+        <link rel="stylesheet" href="./css/styles.css" />
+        <style>
+            .container {
+                background-image: url("./images/Job_background.png");
+                background-repeat: no-repeat;
+                background-position: center;
+                color: rgb(0, 58, 250);
+            }
+            h1,
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 7/home.html]: include in a single line

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

**Comment by sourabh-kanwade** [comment]
changes are done

File: Sourabh_Kanwade/html/Assignment 6/question_2.css [hunk 7]
```diff
@@ -0,0 +1,55 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+h1,h2 {
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 6/question_2.css]: always add space after comma

File: Sourabh_Kanwade/html/Assignment 7/css/styles.css [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+    height: 40px;
+    background-color: #0f376b;
+    color: #fff;
+    font-weight: bold;
+    font-size: 15px;
+    outline: none;
+    border: none;
+    border-radius: 20px;
+}
+form > label,h3 {
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 7/css/styles.css]: add space after comma always

File: Sourabh_Kanwade/html/Assignment 7/home.html [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        <title>Job Portal-Home</title>
+        <link rel="stylesheet" href="./css/styles.css" />
+        <style>
+            .container {
+                background-image: url("./images/Job_background.png");
+                background-repeat: no-repeat;
+                background-position: center;
+                color: rgb(0, 58, 250);
+            }
+            h1,p {
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 7/home.html]: add space after comma

File: Sourabh_Kanwade/html/Assignment 7/personal_details.html [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+            <a href="./personal_details.html" class="sidebar-links">Personal Details</a>
+            <a href="./professional_details.html" class="sidebar-links">Professional Details</a>
+            <a href="./login.html" class="sidebar-links">Login</a>
+        </div>
+        <div class="container">
+            <div class="form-container">
+                <h1>Personal Details</h1>
+                <form>
+                    <label for="first-name">First Name: </label>
+                    <input type="text" name="fname" id="first-name" placeholder="First Name" />
```
Comment (SnehaMakade) [inline Sourabh_Kanwade/html/Assignment 7/personal_details.html]: placeholders second words 1st letter should be small like First name make this change in all input boxes.

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes as per comments given

**Comment by sourabh-kanwade** [comment]
done the changes

**Review by SnehaMakade — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

**Pull request merged**